frontend/src/routes/candidate/(protected)/profile/+page.svelte

```diff
@@ -81,15 +158,15 @@
         <label for="unaffiliated" class={labelClass}>
           {$_('candidateApp.basicInfo.fields.unaffiliated')}
         </label>
-        <input id="unaffiliated" type="checkbox" class="toggle toggle-primary mr-8" checked />
+        <input id="unaffiliated" type="checkbox" class="toggle-primary toggle mr-8" checked />
```

Comment:
These are formatted differently for me. Make sure that `tailwindcss`, `prettier`, and all prettier plugins are up to date (So essentially delete node_modules and reinstall them and hope that fixes it)